File: src/main/java/net/kigawa/kutil/unit/runtimeexception/NoFoundUnitException.java
package net.kigawa.kutil.unit.runtimeexception;

public class NoFoundUnitException extends RuntimeUnitException
{
    public NoFoundUnitException(String message, Throwable cause)
    {
        super(message, cause);
    }

    public NoFoundUnitException(String message)
    {
        super(message);
    }
}

File: kutil-unit-api/src/main/java/net/kigawa/kutil/unitapi/exception/NoFoundUnitException.kt
package net.kigawa.kutil.unitapi.exception

import net.kigawa.kutil.unitapi.UnitIdentify

class NoFoundUnitException(
  override val message: String?,
  override val cause: Throwable? = null,
  override val identify: UnitIdentify<out Any>,
): UnitException()